flipperdevices/flipperzero-firmware
# [FL-1409] Flipper name in reports

# What's new

- USB product string descriptor now formed from flipper name.
- BLE advert name and model string now formed from device name

# Verification 

- Check USB-CDC interface name
- Check BLE advert name and model string

# Checklist (do not modify)
